libpod/util.go

```diff
@@ -235,8 +235,12 @@ func makeInspectPorts(bindings []types.PortMapping, expose map[uint16][]string)
 			for i := uint16(0); i < port.Range; i++ {
 				key := fmt.Sprintf("%d/%s", port.ContainerPort+i, protocol)
 				hostPorts := portBindings[key]
+				var host = port.HostIP
```

Comment:
even though the scope here is minuscule and unambiguous, perhaps `hostIP` would be a clearer variable name

Reply:
done